File: 20140323/fizzbuzz.py
def toStr(num):
    if num % 15 == 0:
        return "FizzBuzz"
    elif num % 3 == 0:
        return "Fizz"
    elif num % 5 == 0:
        return "Buzz"
    else:
        return num

def FizzBuzz(min,max):
    for i in range(min,max):
        print(toStr(i))

FizzBuzz(1,100)

File: 20140523/fizzbuzz.rb
def toStr(num)
  if (num % 15 == 0)
    return 'FizzBuzz'
  elsif (num % 3 == 0)
    return 'Fizz'
  elsif (num % 5 == 0)
    return 'Buzz'
  else
    return num
  end
end

def fizzBuzz(min, max)
  for i in (min..max)
    p toStr(i)
  end
end

fizzBuzz(1, 100)
